website/src/styles/Chakra.tsx

```diff
@@ -1,24 +1,22 @@
-import { ChakraProvider, cookieStorageManagerSSR, localStorageManager } from "@chakra-ui/react";
-
+import { ChakraProvider } from "@chakra-ui/react";
 import { theme } from "./Theme";
 
 export function Chakra({ cookies, children }) {
-  const colorModeManager = typeof cookies === "string" ? cookieStorageManagerSSR(cookies) : localStorageManager;
+  const cookieColorValue = cookies["chakra-ui-color-mode"] ?? "light";
+  const cookieColorModeManager = {
+    type: "cookie",
+    ssr: true,
+    get: () => {
+      return cookieColorValue;
+    },
+    set: () => {
+      console.log("Current colorMode set to :", cookieColorValue);
+    },
+  } as const;
```

Comment:
how it this different from the default `cookieStorageManagerSSR`?

Reply:
I tried passing the cookies to ```cookieStorageManagerSSR``` it didn't work. I don't know why.